compiler/resolution/functionResolution.cpp

```diff
@@ -3514,24 +3534,29 @@ void resolveNormalCall(CallExpr* call) {
   if (call->partialTag && (!best || !best->fn ||
                            !best->fn->hasFlag(FLAG_NO_PARENS))) {
     if (best != NULL) {
-      delete best;
+      // best is deleted below with the other candidates
       best = NULL;
```

Comment:
What does valgrind over the compiler say with this change?  If you're not running into it but it should be a double free, that probably means that whether it is or isn't depends on other circumstances, which worries me.